### PR TITLE
Move Stock Transfers to Stock header

### PR DESCRIPTION
**Pull request by allisonlarson**
Take Stock Transfer management out of Settings, and put it in the Stock
header with stock management.

### CODE REVIEW

**Comment by allisonlarson** [comment]
screen shot 2015-05-15 at 2 37 39 pm

File: backend/app/views/spree/admin/stock_items/index.html.erb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (richardnuno) [inline backend/app/views/spree/admin/stock_items/index.html.erb]: Extra whitespace

**Comment by richardnuno** [comment]
Looks great aside from the whitespace comment :+1:

**Comment by richardnuno** [comment]
Actually just remembered, does this change support the active submenu indicator? Attached an example of how it looks on the product page.
screen shot 2015-05-18 at 11 53 40 am

**Comment by allisonlarson** [comment]
good catch, now 
screen shot 2015-05-18 at 12 52 58 pm

**Comment by richardnuno** [comment]
Still has the extra whitespace in `backend/app/views/spree/admin/stock_items/index.html.erb`, :+1: otherwise

**Comment by allisonlarson** [comment]
:grin:

**Comment by richardnuno** [comment]
:+1:

**Comment by jhawthorn** [comment]
:+1:

**Pull request merged**